packages/client/src/model/update-model-command.ts

```diff
@@ -0,0 +1,126 @@
+/********************************************************************************
```

Comment:
This seems to be a duplicate of the `packages/client/src/base/model/update-model-command.ts`. I assume this was copied by mistake?

Nevertheless, I'm wondering whether we should add the action handler and/or update model command from this file to the glsp-sprotty export as well? At least the `FeedbackAwareUpdateModelCommand` seems to be a specialization of the `UpdateModelCommand`. Thoughts?

Reply:
It is a specialization of the update model command. However, it is only relevant in the glsp context so I don't see a reason to move this into the sprotty export (as it is not replacing any direct sprotty contribution). Also it has dependencies to other GLSP client concepts like the `FeedbackActionDispatcher` and model listeners. We would introduce a dependency circle by moving it into the sprotty export.

Reply:
I have removed the duplicate class